pkg/operator/operators/clustermanager/storageversionmigration/precheck.go

```diff
@@ -0,0 +1,124 @@
+package storageversionmigration
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	migrationv1alpha1 "sigs.k8s.io/kube-storage-version-migrator/pkg/apis/migration/v1alpha1"
+	migrationv1alpha1client "sigs.k8s.io/kube-storage-version-migrator/pkg/clients/clientset/typed/migration/v1alpha1"
+)
+
+var checkList []string = []string{
```

Comment:
@elgnay Please take a look at this checklist, from my understanding, the storageversionmigrations we want to clean the conditions are:
* managedclustersets.cluster.open-cluster-management.io
* managedclustersetbindings.cluster.open-cluster-management.io